packages/server/src/workers/cron.ts

```diff
@@ -216,3 +216,19 @@ export async function removeBullMQJobByKey(botId: string): Promise<void> {
     await queue.removeJobScheduler(botId);
   }
 }
+
+export async function reloadCronBots(): Promise<void> {
+  if (queue) {
+    // Clears all jobs from the cron queue, including active ones
+    await queue.obliterate({ force: true });
```

Comment:
lol at `obliterate`.

obliterate, purge, expunge... we're collecting quite the vocabulary for "kill all the things" 🙃